testFrameworks.json

```diff
@@ -1,4 +1,4 @@
 {
-  "packageNames": ["nock", "jsdom"],
-  "packagePatterns": ["^cypress", "^jest", "^@pollyjs"]
+  "matchPackageNames": ["nock", "jsdom"],
+  "matchPackagePatterns": ["^cypress", "^jest", "^@pollyjs"]
```

Comment:
FYI: the `node`, `orgPackages` and `testFrameworks` presets will currently silently fail due to a bug which I've reported here: https://github.com/renovatebot/renovate/issues/8579